.github/workflows/ci.yml

```diff
@@ -9,144 +9,75 @@ on:
     - reopened
     - synchronize
 jobs:
-  linux-tests:
-    name: linux tests
-    runs-on: ubuntu-18.04
-    env:
-      DISPLAY: ":99"
-      ATOM_GITHUB_BABEL_ENV: coverage
-      MOCHA_TIMEOUT: 60000
-      UNTIL_TIMEOUT: 30000
-    steps:
-    - uses: actions/checkout@v1
-    - name: install Atom
-      shell: bash
-      run: |
-        curl -s -L https://atom.io/download/deb?channel=nightly \
-          -H 'Accept: application/octet-stream' \
-          -o /tmp/atom-amd64.deb
-        /sbin/start-stop-daemon --start --quiet \
-          --pidfile /tmp/custom_xvfb_99.pid \
-          --make-pidfile --background \
-          --exec /usr/bin/Xvfb \
-          -- :99 -ac -screen 0 1280x1024x16
-        sudo apt-get update
-        sudo apt-get install -yyq libgconf-2-4 build-essential git libsecret-1-dev gvfs-bin
-        dpkg-deb -x /tmp/atom-amd64.deb /tmp/atom
-    - name: install dependencies
-      shell: bash
-      run: /tmp/atom/usr/share/atom-nightly/resources/app/apm/bin/apm ci
-    - name: run tests
-      shell: bash
-      run: /tmp/atom/usr/bin/atom-nightly --test test/
-    - name: report code coverage
-      shell: bash
-      env:
-        CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
-        SYSTEM_PULLREQUEST_PULLREQUESTNUMBER: ${{ github.event.number }}
-        SYSTEM_PULLREQUEST_SOURCEBRANCH: ${{ github.head_ref }}
-        BUILD_SOURCEBRANCH: ${{ github.event.ref }}
-      run: |
-        npm run report:coverage
-        bash <(curl -s https://codecov.io/bash) \
-          -n "Linux" \
-          -P "${SYSTEM_PULLREQUEST_PULLREQUESTNUMBER:-}" \
-          -B "${SYSTEM_PULLREQUEST_SOURCEBRANCH:-${BUILD_SOURCEBRANCH}}"
-      if: success() || failure()
-
-  macos-tests:
-    name: macos tests
-    runs-on: macos-10.14
+  tests:
+    name: tests
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, macos-10.14, windows-2016]
+        channel: [stable, beta, nightly]
```

Comment:
I wasn't sure if you only wanted to run against `nightly` or any other channels

Reply:
k